back/internal/usersvc/internal/userdb/mysql/0001_setup_user_table.sql

```diff
@@ -11,7 +11,8 @@ CREATE TABLE user (
     email_hold BOOL NOT NULL DEFAULT TRUE,
     altmail VARCHAR(255) NOT NULL UNIQUE,
     altmail_hold BOOL NOT NULL DEFAULT TRUE,
-    full_name VARCHAR(255) NOT NULL,
```

Comment:
I think we should stick with full name. I've read it is more accessible because not everyone has a strict "first" and "last" name, but everyone has a full name. Perhaps we could add a `nickname` field?

Reply:
I changed it to match what's in the user.proto file. I wonder if the intention here was to combine what was coming from the front end (as `firstName` and `lastName`), and then combine them to put into the database I wonder why that would be, if that's the case. 

I think since most people have a first and last name, we can have both available, but if, like Daved (or even myself), if people don't want to provide a last name, or if they don't have one, the front end doesn't have to require it. I think they can use their nickname in the `firstName` field if they like.

Reply:
Let's keep full_name, please.

Reply:
I managed to get the table fields switched back to full_name, and I even ran `go generate` to get the migration. I had to change the `user.proto` file which still uses the first_name and last_name variables. But, that's where I had problems. I got a lot of errors running protoc. I'll try again tomorrow.

Reply:
With @daved's help, I was able to get the bindata and new pb.go files generate with the change back to using full_name.